spec/features/admin_item_management_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe "Admin Item Management" do
   let(:admin) { create(:admin) }
-  let!(:item) { create(:item) }
+  let!(:listing) { create(:listing) }
```

Comment:
@rshea303 Don't think we need `let!` here. It's better to use `let` when possible.